py/tests/test_proguard.py

```diff
@@ -36,3 +36,20 @@ def test_mmap(res_path):
         view.lookup("android.support.constraint.a.b:a", 116)
         == "android.support.constraint.solver.ArrayRow:createRowDefinition"
     )
+
+
+def test_mapper(res_path):
+    mapper = ProguardMapper.open(os.path.join(res_path, "proguard.txt"))
+    assert mapper.has_line_info
+    assert mapper.uuid == uuid.UUID("5cd8e873-1127-5276-81b7-8ff25043ecfd")
+
+    assert (
+        mapper.remap_class("android.support.constraint.ConstraintLayout$a")
+        == "android.support.constraint.ConstraintLayout$LayoutParams"
+    )
+
+    remapped = mapper.remap_frame("android.support.constraint.a.b", "a", 116)
```

Comment:
Would be nice to have a test that covers inlined frames

Reply:
I added this to the fixture, and tested it ;-)